README.rst

```diff
@@ -225,3 +229,47 @@ Development Speed
 Since this project is relatively stable, time is only dedicated to it on
 Fridays. Thus if you issue a PR, bug, etc, please note that it may take a week
 before we get back to you. Thanks you for your patience.
+
+Development
+-----------
+
+This project uses Poetry to manage dependencies, etc. Thus to install the
+necessary tools when developing, run:
+
+::
+
+    poetry install
+
+Tests
+-----
+
+.. |tests| image:: https://github.com/15five/scim2-filter-parser/workflows/CI%2FCD/badge.svg
+    :target: https://github.com/15five/scim2-filter-parser/actions
+
+https://github.com/15five/scim2-filter-parser/actions
+
+Tests are typically run locally with `tox` (https://tox.wiki/). Tox will test
+all supported versions of Python.
+
+```
+tox
+```
+
+To run the test suite with a single version of Python (the version you created
+the virtualenv with), run:
+
+::
+
+    poetry run pytest tests/
```

Comment:
This doesn't work unless one either installs django manually (not documented here) or if you remove django as a optional dependency from pyproject.toml

Reply:
```
~/misc/scim2-filter-parser on  logston/migtrate-to-poetry ⌚ 11:21:07
$ poetry install
Skipping virtualenv creation, as specified in config file.
Installing dependencies from lock file

Package operations: 19 installs, 1 update, 0 removals

  • Installing attrs (22.1.0)
  • Installing iniconfig (1.1.1)
  • Installing mccabe (0.6.1)
  • Updating packaging (20.9 -> 21.3)
  • Installing pluggy (1.0.0)
  • Installing py (1.11.0)
  • Installing pycodestyle (2.7.0)
  • Installing pyflakes (2.3.1)
  • Installing toml (0.10.2)
  • Installing tomli (2.0.1)
  • Installing coverage (6.4.4)
  • Installing flake8 (3.9.2)
  • Installing isort (4.3.21)
  • Installing pytest (7.1.2)
  • Installing testfixtures (6.18.5)
  • Installing flake8-isort (3.0.1)
  • Installing mock (4.0.3)
  • Installing pytest-cov (3.0.0)
  • Installing sly (0.4)
  • Installing tox (3.25.1)

Installing the current project: scim2-filter-parser (0.4.0)

~/misc/scim2-filter-parser on  logston/migtrate-to-poetry ⌚ 11:21:27
$ which poetry
/home/caleb/.local/bin/poetry

~/misc/scim2-filter-parser on  logston/migtrate-to-poetry ⌚ 11:21:31
$ poetry run pytest tests/
Skipping virtualenv creation, as specified in config file.
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.10.2, pytest-7.1.2, pluggy-1.0.0
rootdir: /home/caleb/misc/scim2-filter-parser, configfile: pytest.ini
plugins: cov-3.0.0
collected 125 items / 1 error                                                                                                                                                                                     

===================================================================================================== ERRORS ======================================================================================================
________________________________________________________________________________ ERROR collecting tests/test_transpiler_django.py _________________________________________________________________________________
ImportError while importing test module '/home/caleb/misc/scim2-filter-parser/tests/test_transpiler_django.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/linuxbrew/.linuxbrew/opt/python@3.10/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_transpiler_django.py:3: in <module>
    from django.db.models import Q
E   ModuleNotFoundError: No module named 'django'
============================================================================================= short test summary info =============================================================================================
ERROR tests/test_transpiler_django.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================================ 1 error in 0.34s =================================================================================================

```

Reply:
Oh shucks, good find.